### PR TITLE
Feature/user org register

### PR DESCRIPTION
**Pull request by Malak67**
- Will drop existing DB and replace with new SCHEMA
- Changed user and org entities/modules/services
- Added new endpoints to user (register, create, update profile, change password, update notifications)
- Removed Role from organization and added it to user (as user can register before having an organization created)
- Fixed linting issues, changed snake_case to camelCase
- Fixed ts errors

### CODE REVIEW

File: apps/drec-api/.eslintrc.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
     project: 'tsconfig.json',
+    tsconfigRootDir: __dirname,
```
Comment (soanvig) [inline apps/drec-api/.eslintrc.js:5]: that helps with something?

**Review by soanvig — APPROVED** [review]
Idon't have strong opinion on these changes, despite minor code issues, so approve

File: apps/drec-api/.eslintrc.js (hunk 1 above)
Comment (Malak67) [reply apps/drec-api/.eslintrc.js:5]: Yep. Eslint was not working properly and this fixed it. Maybe because I am using WSL..not sure. But it fixed it

Comment (soanvig) [reply apps/drec-api/.eslintrc.js:5]: I will try to apply this locally as well

**Pull request merged**